src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -295,7 +335,7 @@ void buildHearingPartiesDetailsJointParty(YesNo jointParty) throws InvalidMappin
     void getIndividualInterpreterLanguageWhenHearingOptionsNull() throws InvalidMappingException {
 
         String individualInterpreterLanguage = HearingsPartiesMapping.getIndividualInterpreterLanguage(
-            null, null, referenceData
+            null, null, referenceData, "French"
```

Comment:
Remember to use data from the sample data or something like "TestLangauge", as to not confuse future devs who might think this is the correct format for the codes